test/logstream/v2/stream.go

```diff
@@ -167,9 +179,27 @@ const (
 	timeFormat = "15:04:05.000"
 	// ChaosDuck is the well known name for the chaosduck.
 	ChaosDuck = "chaosduck"
+	// QueueProxy is the well known name for the queueproxy.
+	QueueProxy = "queueproxy"
 )
 
-func (s *namespaceSource) handleLine(l []byte, pod string) {
+// Names of well known containers that do not produce nicely formatted logs that
+// could be easily filtered and parsed by handleLine. Logs from these containers
+// are captured without filtering.
+func getWellKnownContainers() []string {
```

Comment:
```suggestion
func 
wellKnownContainers() []string {
```

But more importantly why this isn't a variable (`sets.String` seems to help to avoid lookup in the isWKC below)